images/build.sh

```diff
@@ -41,16 +43,35 @@ if [[ -z "${CONTAINER_ENGINE:-}" ]]; then
     fi
 fi
 
+ensure_buildx() {
+    ${CONTAINER_ENGINE} buildx rm multiarch || true
+    ${CONTAINER_ENGINE} buildx create --use --name=multiarch --bootstrap --platform=${multi_arch_platforms}
+}
+
 image_build() {
     local cmd=$1
     local image_dir
+    local enable_buildx=${2:-}
+    local container_engine_build="${CONTAINER_ENGINE} build"
     if [[ -d ./images/"${cmd}" ]]; then
         image_dir="${cmd}"
     else
         image_dir="default"
     fi
+
+    if [[ "${enable_buildx}" == 1 ]]; then
+        if [[ "${CONTAINER_ENGINE}" == "podman" ]]; then
+            echo "Podman is not preferred when ENABLE_BUILDX is set, terminating the build!"
+            exit 1 
+        else
+            ensure_buildx
+            echo "ENABLE_BUILDX is set! Building multi-arch image for '${cmd}' for platforms '${multi_arch_platforms}'" >&2
+            container_engine_build="${CONTAINER_ENGINE} buildx build --builder multiarch --platform=${multi_arch_platforms} --output type=registry"
```

Comment:
Added `--output type=registry` since `--output type=docker` or `--load` is not supported which stores the built images on local when two platforms are build simultaneously and errors with
`ERROR: docker exporter does not currently support exporting manifest lists`

If this change is okay, should the multi arch images be removed from the list here?https://github.com/kubernetes-sigs/boskos/blob/master/images/cloudbuild.yaml#L40

Reply:
Can we build multi arch by default? build amd64,arm64 and ppc64le

Reply:
@upodroid, added changes, PTAL.